src/gridcoin/staking/status.cpp

```diff
@@ -18,6 +18,7 @@ namespace {
 //!
 constexpr const char* STAKING_ERROR_STRINGS[] {
     "None",
+    "Commanded disabled",
```

Comment:
```suggestion
    "Disabled by configuration",
```

Technically accurate, but, personally, I'd have no clue what that meant 🙂